src/main/java/seedu/duke/data/TaskList.java

```diff
@@ -57,6 +64,14 @@ public Task addTag(String tagDescription, int index) throws NoSuchTaskException
         return task;
     }
 
+    /**
+     * Removes tag from the task list.
+     *
+     * @param tagDescription The description of tag that is inputted by user
+     * @param index The task number to remove the tag
```

Comment:
Index isn't really the same as task number. Maybe change the description to `Index of task to be added with tag.`